lib/nimble_template/addons/variants/phoenix/web/svg_sprite.ex

```diff
@@ -35,7 +35,7 @@ defmodule NimbleTemplate.Addons.Phoenix.Web.SvgSprite do
       """,
       """
         "scripts": {
-          "svg-sprite.generate-icon": "svg-sprite --shape-id-generator \\"icon-%s\\" --symbol --symbol-dest static/images --symbol-sprite icon-sprite.svg static/images/icons/*.svg",
+          "svg-sprite.generate-icon": "svg-sprite --shape-id-generator \\"icon-%s\\" --symbol --symbol-dest ../priv/static/images --symbol-sprite icon-sprite.svg ../priv/static/images/icons/*.svg",
```

Comment:
You removed this [here](https://github.com/nimblehq/elixir-templates/pull/197/files#diff-86afafd7ad9b4533f3c5b9a2ac31f04c8fca1ecc270dc9f8f8e6a504f7f52105R38) and added it back? 😂

Reply:
hmm, maybe because the git conflict 🤔

Reply:
solved @bterone